multiqc/modules/rsem/rsem.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from Cutadapt """
+
+from __future__ import print_function
+import logging
+import re
+from distutils.version import StrictVersion
+from collections import OrderedDict
+
+from multiqc import config
+#from multiqc.plots import linegraph
+from multiqc.plots import bargraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+    Cutadapt module class, parses stderr logs.
+    Also understands logs saved by Trim Galore!
+    (which contain cutadapt logs)
+    """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Rsem', anchor='rsem',
+        href='https://deweylab.github.io/RSEM/',
+        info="RSEM (RNA-Seq by Expectation-Maximization) is a software package for"\
+             "estimating gene and isoform expression levels from RNA-Seq data.")
+    
+         # Find and load any featureCounts reports
+        self.rsem_mappable_data = dict()
+        self.rsem_mapped_data = dict()
+        self.rsem_keys  = list()
```

Comment:
I don't think you really need this list. It seems to only be used for debugging. Also the same list can be obtained easily with `self.rsem_mapped_data.keys()` if you need it anyway.